app/src/main/res/xml/preferences_nextcloud_gpodder.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen
+        xmlns:android="http://schemas.android.com/apk/res/android">
+    <Preference
+            android:key="pref_nextcloud_gpodnet_description"
+            android:icon="@drawable/nextcloud_logo_svg"
+            android:summary="Use Nextcloud GPodder app to sync"/>
```

Comment:
Please use a string resource. Also, I think it would be useful to describe how to get the app and that you need to self-host it.